test/src/module/integration/allTest.c

```diff
@@ -44,6 +44,9 @@ Test statuses
 
 #define TEST_RESTORE_POINT                                          "pgbackrest"
 
+#define UPDATE_SPOOL_PATH() \
+    hrnHostConfigUpdateP(.archiveAsync = BOOL_TRUE_VAR, .spoolPathPostfix = VARSTR(STRDEF(STRINGIFY(__COUNTER__))))
+
```

Comment:
I'm not sure how I feel about using a GCC extension here. In the core code this would definitely not be OK, but even in the tests I feel like we should avoid it. If we build the logic into TEST_HOST_BR() (see below) I think the macro is less compelling overall.